app/apphandlers/processorAppHandler.go

```diff
@@ -267,6 +268,13 @@ func (processor *ProcessorApp) StartRudderCore(ctx context.Context, options *app
 		return rsourcesService.CleanupLoop(ctx)
 	})
 
+	g.Go(func() error {
+		replicationLagStat := stats.DefaultStats.NewStat("rsources_log_replication_lag", stats.GaugeType)
+		replicationSlotStat := stats.DefaultStats.NewStat("rsources_log_replication_slot", stats.GaugeType)
```

Comment:
won't it be better if we move this to `monitor` func, instead of declaring it here and then needlessly passing them as arguments?

Reply:
didn't want to import the entire package over there.
Something that `Gauge`s is all we need inside the package under consideration.